LayoutTests/interaction-region/inline-link.html

```diff
@@ -7,6 +7,7 @@
 <a href="#">This</a> is a link.
 <a href="#" style="border: 0; background: none;">This</a> is a wiki-style link.
 <a href="#" style="background: green;">This</a> is a link with visible edges.
+<a href="#" style="text-decoration: none; border-bottom: 1px blue solid;">border-bottom link</a>
```

Comment:
This seems insufficient test coverage. Look at all the different checks that are needed to get this right. We should add more than one additional test case for the various combinations rather than just the one that we got a bug report on.

Reply:
Can add these after landing this, though.